_vector-search/ai-search/hybrid-search/collapse.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: default
+title: Using collapse with a hybrid query
+parent: Hybrid search
+grand_parent: AI search
+has_children: false
+nav_order: 10
+---
+
+# Using collapse with hybrid query
+**Introduced 3.1**
+{: .label .label-purple }
+
+You can collapse on a field by specifying the `collapse` parameter in the search query.
+This will return the highest scoring document for each unique value of that field, up to the specified size in the query.
+The collapse parameter requires the field being collapsed to be of either a `keyword` or a `numeric` type.
```

Comment:
please add what will happen if field is neither keyword nor numeric, and if that's the same for other query types (I assume we follow general pattern for supported field types)